packages/snaps-execution-environments/jest.environment.js

```diff
@@ -13,6 +13,7 @@ module.exports = class CustomTestEnvironment extends TestEnvironment {
       this.global.TextDecoder = TextDecoder;
       this.global.ArrayBuffer = ArrayBuffer;
       this.global.Uint8Array = Uint8Array;
+      this.global.harden = (param) => param;
```

Comment:
Were Jest tests failing with the real `harden`?

Reply:
Yes, the Jest was failing with `ReferenceError: harden is not defined` when that mock was not in the place.
Following your suggestion we decided to use only AVA for the tests that use `harden`.